core/src/main/java/com/backblaze/b2/client/credentialsSources/B2Credentials.java

```diff
@@ -9,9 +9,9 @@
  */
 public interface B2Credentials {
     /**
-     * @return the accountId to use for b2_authorize_account.
+     * @return the applicationKeyId to use for b2_authorize_account.
      */
-    String getAccountId();
+    String getApplicationKeyId();
```

Comment:
this is an incompatible API change, methinks.  with this change, i think we will need to bump the major version number of the SDK.  is that necessary?

Reply:
Can you help me understand why this is an incompatible API change?